Code/PgSQL/rdkit/Makefile

```diff
@@ -47,7 +47,7 @@ SHLIB_LINK += -pthread
 
 SHLIB_LINK += ${THREADLIBS}
 
-PG_CPPFLAGS = -I${BOOSTHOME}/include -I${RDKIT}/Code -DRDKITVER='"007300"' ${AVALONFLAGS} ${INCHIFLAGS} ${POPCOUNTFLAGS}
+PG_CPPFLAGS = -I${BOOSTHOME}/include -I${RDKIT}/Code -DRDKITVER='"007300"' ${AVALONFLAGS} ${INCHIFLAGS} -DUSE_BUILTIN_POPCOUNT ${POPCOUNTFLAGS}
```

Comment:
yes, I think that would make more sense. it would allow that to be turned off, which will otherwise (I think) be impossible.